lazy_tensor_core/lazy_bench.py

```diff
@@ -211,6 +213,7 @@ def iter_models(args):
             or name in SKIP
             or (name in SKIP_TRAIN_ONLY and args.test == "train")
         ):
+            save_error(name, args.test, "in SKIP or SKIP_TRAIN_ONLY", dirpath)
```

Comment:
do we have a consistent place where we save_error generally?  e.g. is it sometimes in iter_models and sometimes in the experiment code?  i guess it has to be in iter_models in this case.

Reply:
I think it needs to be called in multiple places:
* when we filter out models
* when a time out is triggered
* in lazy_compute, lazy_overhead for the rest of the errors

since we are dumping errors in files in subprocesses, it could be a bit tricky to have just one place for `save_error`.

Reply:
makes sense.  it would be nice if the 'rest of the errors' could be caught from one place near where timeout is triggered. but i am ok with the way it is.